CMakeLists.txt

```diff
@@ -1,10 +1,18 @@
 cmake_minimum_required(VERSION 3.10)
 
 file(GLOB SOURCES src/littlefs/*.c)
-list(APPEND SOURCES src/esp_littlefs.c src/littlefs_api.c src/lfs_config.c)
+list(APPEND SOURCES src/esp_littlefs.c src/littlefs_esp_part.c src/lfs_config.c)
+
+if(CONFIG_LITTLEFS_SDMMC_SUPPORT)
+    list(APPEND SOURCES src/littlefs_sdmmc.c)
+endif()
 
 if(IDF_VERSION_MAJOR GREATER_EQUAL 5)
-    list(APPEND pub_requires esp_partition)
+    if (CONFIG_LITTLEFS_SDMMC_SUPPORT)
+        list(APPEND pub_requires esp_partition sdmmc)
+    else()
+        list(APPEND pub_requires esp_partition sdmmc)
```

Comment:
Oh yes, my bad, fixing now

Reply:
Odd, this doesn't work: 

![image](https://github.com/joltwallet/esp_littlefs/assets/4463497/8b115910-939f-47c9-ab1e-62165d78727c)

It will result in `fatal error: sdmmc_cmd.h: No such file or directory` since somehow the `sdmmc` library isn't included. I've tried rebuilding after a full clean but problem still presists.